### PR TITLE
Minor performance

### PR DESCRIPTION
**Pull request by luis-pereira**

### CODE REVIEW

File: xdgdesktopfile.cpp [hunk 1]
```diff
@@ -935,7 +935,7 @@ QString expandEnvVariables(const QString str)
 QStringList expandEnvVariables(const QStringList strs)
 {
     QStringList res;
-    foreach(QString s, strs)
+    foreach(const QString s, strs)
```
Comment (paulolieuthier) [inline xdgdesktopfile.cpp]: Missed `&` here?

Comment (luis-pereira) [inline xdgdesktopfile.cpp]: yup

**Comment by paulolieuthier** [comment]
GTM.

**Comment by palinek** [comment]
GTM

**Pull request merged**